pages/about/index.js

```diff
@@ -40,7 +40,7 @@ export default function About() {
         </div>
         <div className={styles.about__image}>
           <Image
-            src="/ufo-team.png"
+            src="/about-us.png"
```

Comment:
nit: instead of creating a new image with a new name, could you just replace existing one.
So under the `ufo-team.png` upload compressed image

Reply:
Done